roles/app/templates/postgresql.yml.j2

```diff
@@ -33,3 +36,4 @@ production:
   username: {{ db_user }}
   password: {{ db_password }}
   host: {{ db_host }}
+  port: {{ db_port }}
```

Comment:
I wish someday we could rely on ENV vars alone and not have this duplicate as a template and as yml in the app :disappointed: